_rest_config.php

```diff
@@ -254,6 +254,11 @@ static function authentication_check($resource)
             }
         }
     }
+
+    static function apiLog($requestUrl, $response, $requestBody = '')
+    {
+        sqlStatementNoLog("INSERT INTO `api_log` (`user_id`, `patient_id`, `request_url`, `request_body`, `response`, `created_time`) VALUES (?, ?, ?, ?, ?, NOW())", [($_SESSION['authUserID'] ?? 0), ($_SESSION['pid'] ?? 0), $requestUrl, $requestBody, $response]);
```

Comment:
I'd add IP for phishers.
request bodys can get quite large. Maybe request type i.e login, auth, request action. Unsure what we gain with logging unless there's some intelligence built in to do something with the log or debugging.  Still it's a little closer to a review process which I still feel strongly about.